hofund-core/src/main/java/dev/logchange/hofund/connection/HofundConnectionsTable.java

```diff
@@ -14,10 +15,13 @@ public class HofundConnectionsTable {
 
     public HofundConnectionsTable(List<HofundConnectionsProvider> connectionsProviders) {
         List<HofundConnection> connections = new ArrayList<>();
+
         for (HofundConnectionsProvider connectionsProvider : connectionsProviders) {
             connections.addAll(connectionsProvider.getConnections());
         }
 
+        connections.sort((d1, d2) -> d2.getType().compareTo(d1.getType()));
```

Comment:
@marwin1991 

Maybe better way??

```java
this.connections = connectionsProviders.stream()
				.map(HofundConnectionsProvider::getConnections)
				.sorted((d1, d2) -> d2.getType().compareTo(d1.getType()))
				.collect(Collectors.toList())
```

Reply:
@Athi I am not a fun of streams xD

Reply:
@Athi but I applied your change with small imporvemnt to use flatMap (it's list of lists)